vendor/github.com/mongodb/mongo-tools-common/util/exit_code.go

```diff
@@ -11,13 +11,23 @@ import (
 )
 
 const (
-	ExitError      int = 1
-	ExitClean      int = 0
-	ExitBadOptions int = 3
-	ExitKill       int = 4
-	// Go reserves exit code 2 for its own use
+	ExitFailure int = 1
```

Comment:
nit: I think this would be cleaner using `iota`. Eg:
```
const (
    ExitSuccess int = iota
    ExitFailure
)
```

Reply:
done